build.sbt

```diff
@@ -50,6 +50,7 @@ lazy val dockerHashing = project
 
 lazy val cromwellApiClient = project
   .withLibrarySettings("cromwell-api-client", cromwellApiClientDependencies)
+  .dependsOn(cwl)
```

Comment:
while certainly an improvement over depending on `engine` I'm still not sure conceptually why the API client should need this? can this be just the `centaurCwlRunner` that needs `cwl`?

Reply:
:man_facepalming: .  I didn't realize this was in the wrong place!